.github/workflows/abtesting.yml

```diff
@@ -28,18 +28,32 @@ jobs:
         uses: actions/checkout@master
       - name: Setup
         run: |
-          cd abtesting
+          cd abtesting/LegacyABTestingQuickstart
           gem install bundler
           bundle install
           gem install xcpretty
           bundle exec pod install --repo-update
+          cd ..
           ../scripts/install_prereqs/abtesting.sh
         env:
           SAMPLE: ABTesting
           secrets_passphrase: ${{ secrets.GHASECRETSGPGPASSPHRASE1 }}
+          LEGACY: true
       - name: Build Swift
         run: |
           ./scripts/test.sh
         env:
-          SWIFT_SUFFIX: ''
+          SAMPLE: ABTesting
+          LEGACY: true
+      - name: Build SwiftUI
+        run: |
+          cd abtesting
+          ../scripts/install_prereqs/abtesting.sh
+          xcodebuild \
+          -project 'ABTestingExample.xcodeproj \
```

Comment:
The build is failing because this line is missing a closing `'`.

Reply:
Done ✅